JSDemos/Demos/Charts/AxisLabelsTemplates/description.md

```diff
@@ -0,0 +1,5 @@
+Axes [labels](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/label/) are visual elements that display values of the [major axis ticks](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/tick/).
+
+To configure labels, specify their settings in the **label** object of the [value](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/label/) or [argument](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/label/) axis. To specify options for all axes, use the **commonAxisSettings**.[label](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/label/) object. Individual settings take precedence over these settings.
+
+This demo shows how to implement labels with custom content. To do this, declare the SVG markup in the [template](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/label/#template) option. You can access the label's raw or converted to string data inside the template via its parameters. 
```

Comment:
> converted to string data

Can we use "formatted value" instead?

> inside the template via its parameters

"Parameters" can be only in functions, which is all about jQuery. How about we rewrite this sentence as follows:

"You can access a label's raw and formatted values in template data."